browser/templates/browser/change_person.html

```diff
@@ -1,7 +1,13 @@
 {% extends "browser/change_base.html" %}
 {% load addcss %}
 
-{% block title %}Editing: {{ person.name }} (Person){% endblock %}
+{% block title %}
+{% if add_person %}
```

Comment:
just check if 'person' is set and remove this variable.

Reply:
Changed to if 'person'